RxJavaTemplate/app/src/test/java/co/nimblehq/rxjava/ui/screens/home/HomeViewModelTest.kt

```diff
@@ -20,45 +20,53 @@ class HomeViewModelTest {
     @Before
     fun setup() {
         When calling mockGetExampleDataUseCase.execute(any()) itReturns Single.just(MockUtil.data)
-        viewModel = HomeViewModel(
-            mockGetExampleDataUseCase
-        )
+        viewModel = HomeViewModel(mockGetExampleDataUseCase)
     }
 
     @Test
-    fun `When initializing, it should emit first data loading`() {
+    fun `When initializing HomeViewModel, it emits the first data correspondingly`() {
```

Comment:
the viewmodel name could be in a common name for all test classes
```suggestion
    fun `When initializing the view model, it emits the first data correspondingly`() {
```